examples/permissioned-token/scripts/common/common.js

```diff
@@ -10,23 +10,30 @@ exports.executeTransaction = async function (deployer, txnParams) {
 };
 
 /**
- * Fund an account from master with some algos (= 20)
+ * Fund accounts from master with 20 Algos
  * @param {*} deployer algobDeployer
- * @param {*} account account to fund algos(20)
+ * @param {*} account or list of accounts to fund
  */
 exports.fundAccount = async function (deployer, account) {
   const master = deployer.accountsByName.get('master-account');
-
-  try {
-    console.log(`* Funding Account:${account.name} *`);
-    await executeTransaction(deployer, {
+  const params = [];
+  if (!(account instanceof Array)) {
+    account = [account];
+  }
+  for (const a of account) {
+    console.log(`* Funding Account: ${a.name} *`);
+    account.push({
       type: types.TransactionType.TransferAlgo,
       sign: types.SignType.SecretKey,
       fromAccount: master,
-      toAccountAddr: account.addr,
+      toAccountAddr: a.addr,
       amountMicroAlgos: 20e6,
       payFlags: { totalFee: 1000, note: 'funding account' }
     });
+  }
```

Comment:
here we're pushing in `account` array while looping  on account.

Reply:
this should be `params.push`. Right?

Reply:
yes, need to run all of this. Updated. Thanks for spotting.